doc/source/tune/_tutorials/tune-sklearn.py

```diff
@@ -0,0 +1,147 @@
+# flake8: noqa
+"""
+Tune's Scikit Learn Adapters
+============================
+
+Scikit-Learn is one of the most widely used tools in the ML community for working with data, offering dozens of easy-to-use machine learning algorithms. However, to achieve high performance for these algorithms, you often need to perform **model selection**.
+
+
+.. image:: /images/tune-sklearn.png
+    :align: center
+    :width: 50%
+
+Scikit-Learn `has an existing module for model selection <https://scikit-learn.org/stable/modules/grid_search.html>`_, but the algorithms offered (grid search and random search) are often considered ineffifient. In this tutorial, we'll cover ``tune-sklearn``, a drop-in replacement for Scikit-Learn’s model selection module with state-of-the-art optimization features such as early stopping and Bayesian Optimization.
+
+Check out the repo here: https://github.com/ray-project/tune-sklearn.
+
+Overview
+--------
+
+``tune-sklearn`` is a module that integrates Ray Tune's hyperparameter tuning and scikit-learn's Classifier API. It is a drop-in replacement for GridSearchCV and RandomizedSearchCV, so you only need to change less than 5 lines in a standard Scikit-Learn script to use the API.
+
+``tune-sklearn`` allows you to easily leverage Bayesian Optimization, HyperBand, and other cutting edge tuning techniques by simply toggling a few parameters. It also supports and provides examples for many other frameworks with Scikit-Learn wrappers such as Skorch (Pytorch), KerasClassifiers (Keras), and XGBoostClassifiers (XGBoost).
+
+Run ``pip install ray[tune] tune-sklearn`` to get started.
+
+Walkthrough
+-----------
+
+``tune-sklearn`` has two APIs: TuneSearchCV, and TuneGridSearchCV. They are drop-in replacements for Scikit-learn's RandomizedSearchCV and GridSearchCV.
+
+For this example, let’s start by using ``TuneGridSearchCV`` with sklearn’s `digits dataset`_ and a `SGDClassifier`_ to classify digits.
+
+.. _`digits dataset`: https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_digits.html
+.. _`SGDClassifier`: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html
+
+To start out, change the import statement to get tune-scikit-learn’s grid search cross validation interface:
+
+"""
+# from sklearn.model_selection import GridSearchCV
+from tune_sklearn import TuneGridSearchCV
```

Comment:
```suggestion
from tune_sklearn.tune_gridsearch import TuneGridSearchCV
```

Reply:
```
>>> from tune_sklearn.tune_gridsearch import TuneGridSearchCV
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'tune_sklearn.tune_gridsearch'

Reply:
Did you update your tune-sklearn? We have a new version on pip that includes our refactoring

Reply:
Requirement already up-to-date: tune-sklearn in /Users/rliaw/miniconda3/lib/python3.7/site-packages (0.0.1)

Can you bump the version?

Reply:
Yeah we need to put this into pip; there's a small change we'd also like to group into this version but it hasn't been merged in yet. The newest version will be put onto pip when we get it merged.